example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -69,6 +82,112 @@ async function setupCollectionAndTestData(config, testSuffix, testDataFolder) {
   ]);
 }
 
+/**
+ * Creates a new test collection with associated granule for testing.
+ *
+ * @param {string} prefix - stack Prefix
+ * @param {string} sourceBucket - testing source bucket
+ * @returns {Promise<Object>} new collection Object
+ */
+const createActiveCollection = async (prefix, sourceBucket) => {
+  // The S3 path where granules will be ingested from
+  const sourcePath = `${prefix}/tmp/${randomId('test-')}`;
+
+  // Create the collection
+  const newCollection = await createCollection(
+    prefix,
+    {
+      duplicateHandling: 'error',
+      process: 'modis',
+    }
+  );
+
+  // Create the S3 provider
+  const provider = await createProvider(prefix, { host: sourceBucket });
+
+  // Stage the granule files to S3
+  const granFilename = `${randomId('junk-file-')}.txt`;
+  const granFileKey = `${sourcePath}/${granFilename}`;
+  await s3PutObject({
+    Bucket: sourceBucket,
+    Key: granFileKey,
+    Body: 'aoeu',
+  });
+
+  const granuleId = randomId('granule-id-');
+
+  const ingestTime = Date.now() - 1000 * 30;
+
+  // Ingest the granule.
+  const testExecutionId = randomId('test-execution-');
+  console.log('testExecutionId:', testExecutionId);
+  const ingestGranuleRule = await createOneTimeRule(
```

Comment:
Why are we creating a rule here versus just running the workflow?

Reply:
This was the way Mark B suggested as the "refactored" approach to ingesting a granule. And I wasn't 100% sure the testDataFolder could be set up for a new collection/granule.

Also I liked this could be pulled out and used elsewhere if we wanted since it ingests the granule but also passes back the cleanup function that needs to be called to remove all of the setup pieces after the test is run. 
But if you think it's confusing, I can probably figure out how to setupTestGranuleForIngest for this test case.

Reply:
Well, it's not that I think it's confusing. I do think it adds extra latency to our tests and also the one time rule test is one that we have intermittent fails on (I've been playing around with it when I have time), so this to me has a high chance of failing intermittently.

I like the cleanup function idea and think it would be a good idea to move ingesting a granule into a helper function with cleanup.

Reply:
After looking at this, I couldn't see how to easily use the existing ingestAndPublishGranule, so compromised with creating the collection/provider/testObject and running buildAndExecuteWorkflow in the createActiveCollection routine.  I parallelized the call to be done at the same time the other granules are being ingested, and parallelized the cleanup.  I hope this addresses your concerns. This PR is ready for re-review. @laurenfrederick